### DISCUSSION TITLE
complete unblocking

### DISCUSSION THREAD

**Discussion by yuTool666** [3kh0/ext-remover#1037] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
*It may not be possible with version 123 or later.
1.  First, do a power wash.
2.  Next, connect wifi and go to the screen where you enter your account password
3.  Enter your password, press enter, and count for 6 seconds.
4.  Turn off the wifi as soon as you finish counting
5.  Check "chrome://extensions" and if no extensions are installed, everything is fine.
6.  Press ctrl + alt + t to open crosh
7.  Then enter "vmc create-extra-disk --size 1 /home/chronos/user/Extensions"
      If an error occurs, start over from scratch
8.   When something is displayed in white text, it is complete. Please connect to wifi.

**Comment by CGRmags** [comment]
they let you do crosh lol thats insane your tech department is dead

**Reply by Inglan2** [reply]
I think turning off wifi prevents the policy from being loaded

**Reply by Aka-but-nice** [reply]
Yeah that is not how it works. DNS or WiFi is not taken into consideration on policy load. Even if it did work there's not much you can do with it, once you turn Wi-Fi on policies would simply reinstitute. Although it's a practical useless idea, keep trying, who knows this could be the next watered down sh1mmer.

**Reply by Inglan2** [reply]
I mean the policy can't be pulled from dm.google.com or whatever. you could also probably connect to your home network and run an Adguard home server to block that domain. I can find the whole domain later today when I get home.

**Reply by Inglan2** [reply]
maybe the `vmc create-extra-disk --size 1 /home/chronos/user/Extensions` would do something about the policy. if you could turn devmode on and use fakemurk or something you could set local device policies probably

**Comment by myschoolisass** [comment]
basically corkey but it works indefinately and you need crosh

**Comment by gomes-web** [comment]
Instead of using crosh, can this work with developer tools?

**Comment by Aka-but-nice** [comment]
this.is.just.vk6s.exploit
